CMakeLists.txt

```diff
@@ -50,8 +50,14 @@ message(STATUS "Build type is ${CMAKE_BUILD_TYPE}")
 enable_testing(true)
 
 if(CMAKE_CXX_COMPILER_ID MATCHES "GNU" OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
-    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -std=c99 -Wall -Werror -fdiagnostics-show-option")
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAS} -std=c++11 -Wall -Werror -fdiagnostics-show-option")
+	if(WERROR MATCHES OFF)
+		message("-> switching off warning-errors. make sure you everything if fine with -DWERROR=ON once you are done hacking around. ")
+		set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -DNOFLIP_BOUNDS -std=c99 -Wall -fdiagnostics-show-option")
```

Comment:
how did the NOFLIPBOUNDS flag appear here again? We had eliminated from the master branch

Reply:
my bad. ea9800 fixes this.